### DISCUSSION TITLE
Limiting FPS?

### DISCUSSION THREAD

**Discussion by Ilham-Habibullin** [zed-industries/zed#8869] [Feature Requests]
Is there an option to limit maximum FPS? Like in computer games?

If i don't need 120 or 60 and for me 30 is enough and i don't want to create extra load on my computer so that it won't heat would it make sense to limit FPS?

**Comment by Moshyfawn** [comment]
There's no native FPS cap in Zed.

From what I've noticed, the MacOS low-power mode, when enabled, tends to limit the FPS of applications.

**Comment by woboti** [comment]
Well, while that may be OK, I don't believe it to be a sure fire way.
Also, it may mitigate some issues likely stemming from #15166 such as #14074 and #23479

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
